packages/core/helper-plugin/src/hooks/useSelectionState.ts

```diff
@@ -1,9 +1,16 @@
 import { useState } from 'react';
 
-export const useSelectionState = (keys, initialValue) => {
+type JSON = string | number | boolean | null | { [key: string]: JSON } | Array<JSON>;
+
+export type TValues = Record<string, JSON>;
+
+export const useSelectionState = <TValues extends object>(
+  keys: (keyof TValues)[],
```

Comment:
```suggestion
  keys: Array<keyof TValues>,
```

remove those `(` can break the hooks, so lets not let that be so easy 👍🏼